modules/workbench/modules/monitoring/modules/metric_descriptors/assets/monitoring_metric_descriptors/billing_buffer_project_count_3.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I couldn't delete the original billing_buffer_project_count.json because it's "still in use" - though I couldn't figure out where